src/midiEditorCommands.cpp

```diff
@@ -994,6 +994,55 @@ void cmdMidiFilterWindow(Command *command) {
 	}
 }
 
+void maybePreviewCurrentNoteInEventList(HWND hwnd) {
+	auto focused = ListView_GetNextItem(hwnd, -1, LVNI_FOCUSED);
+	char text[50];
+	// Get the text from the type column (4).
+	ListView_GetItemText(hwnd, focused, 4, text, sizeof(text));
+	if (strcmp(text, "Note") != 0) {
```

Comment:
While OSARA itself isn't translatable, I try to avoid any code which depends on REAPER's own localised strings. Is there some other way we can detect this? For example, I'm guessing notes are the only event type that have a length value?

Reply:
I will look in to this, good question.